### PR TITLE
CLOUD-282-doc-404

### PR DESCRIPTION
**Pull request by patrickbirch**
Updated link and RST format for Kubernetes.rst and conf.py.

### CODE REVIEW

File: source/kubernetes.rst [hunk 1]
```diff
@@ -1,44 +1,46 @@
...
-   .. code:: bash
+   .. code-block:: bash
 
-      git clone -b release-0.3.0 https://github.com/percona/percona-xtradb-cluster-operator
+      git clone -b release-1.0.0 https://github.com/percona/percona-xtradb-cluster-operator
       cd percona-xtradb-cluster-operator
 
-   **Note:** *It is crucial to specify the right branch with ``-b``
-   option while cloning the code on this step. Please be careful.*
+   **Note:** *You must specify the correct branch with ``-b``
```
Comment (delgod) [inline source/kubernetes.rst]: it is really crucial, it is the most popular bug, so the user should be careful here.

File: source/kubernetes.rst [hunk 2]
```diff
@@ -49,42 +51,44 @@ Install Percona XtraDB Cluster on Kubernetes
...
+   .. code-block:: bash
 
       $ kubectl apply -f deploy/operator.yaml
 
-4. Now that’s time to add the PXC Users secrets to Kubernetes. They
-   should be placed in the data section of the ``deploy/secrets.yaml``
-   file as base64-encoded logins and passwords for the user accounts
+4. Now that’s time to add the |Percona XtraDB Cluster| Users secrets to Kubernetes. They
+   should be placed in the data section of :file:``deploy/secrets.yaml``
+   as logins and base64-encoded passwords for the user accounts
```
Comment (delgod) [inline source/kubernetes.rst]:
```suggestion
   as base64-encoded passwords for the user accounts
```

File: source/kubernetes.rst [hunk 3]
```diff
@@ -49,42 +51,44 @@ Install Percona XtraDB Cluster on Kubernetes
...
 
-   **Note:** *the following command can be used to get base64-encoded
-   password from a plain text string:*
+   **Note:** *the following command decodes the base64-encoded
+   password:*
    ``$ echo -n 'plain-text-password' | base64``
 
-   After editing is finished, users secrets should be created (or
-   updated with the new passwords) using the following command:
+   After editing the user name and password information the user secrets are created (or
```
Comment (delgod) [inline source/kubernetes.rst]:
```suggestion
   After editing the password information the user secrets should be created (or
```

**Comment by patrickbirch** [comment]
Resolved conflicts

**Comment by fiowro** [comment]
Fixed by two other pull requests, https://github.com/percona/percona-xtradb-cluster-operator/pull/235 and https://github.com/percona/percona-xtradb-cluster-operator/pull/233

**Pull request closed without merging**